examples/systemcalls/systemcalls.c

```diff
@@ -58,12 +145,53 @@ bool do_exec(int count, ...)
  *   as second argument to the execv() command.
  *
 */
-
+    if(check_abs_path(command[0])==false)
+    {
+        syslog(LOG_ERR, "Path must be an absolute path \n");
```

Comment:
I don't think it's strictly required to use an absolute path, a path relative to the current working directory could also theoretically be supported.